src/scripts/main.js

```diff
@@ -1,3 +1,428 @@
+
+const optionMakeChoise = document.createElement('option');
+
+optionMakeChoise.setAttribute('value', '');
+optionMakeChoise.innerHTML = '-- Choose a city --';
+officeInput.append(optionMakeChoise);
+
+const optionTokyo = document.createElement('option');
+
+optionTokyo.setAttribute('value', 'Tokyo');
+optionTokyo.innerHTML = 'Tokyo';
+officeInput.append(optionTokyo);
+
+const optionSingapore = document.createElement('option');
+
+optionSingapore.setAttribute('value', 'Singapore');
+optionSingapore.innerHTML = 'Singapore';
+officeInput.append(optionSingapore);
+
+const optionLondon = document.createElement('option');
+
+optionLondon.setAttribute('value', 'London');
+optionLondon.innerHTML = 'London';
+officeInput.append(optionLondon);
+
+const optionNewYork = document.createElement('option');
+
+optionNewYork.setAttribute('value', 'New York');
+optionNewYork.innerHTML = 'New York';
+officeInput.append(optionNewYork);
+
+const optionEdinburgh = document.createElement('option');
+
+optionEdinburgh.setAttribute('value', 'Edinburgh');
+optionEdinburgh.innerHTML = 'Edinburgh';
+officeInput.append(optionEdinburgh);
+
+const optionSanFrancisco = document.createElement('option');
+
+optionSanFrancisco.setAttribute('value', 'San Francisco');
+optionSanFrancisco.innerHTML = 'San Francisco';
+officeInput.append(optionSanFrancisco);
+
+const ageLabel = document.createElement('label');
+
+ageLabel.innerHTML = 'Age:';
+ageLabel.setAttribute('for', 'age');
+form.append(ageLabel);
+
+const ageInput = document.createElement('input');
+
+ageInput.setAttribute('type', 'number');
+ageInput.setAttribute('data-qa', 'age');
+ageInput.setAttribute('id', 'age');
+ageInput.setAttribute('required', '');
+ageLabel.append(ageInput);
+
+const salaryLabel = document.createElement('label');
+
+salaryLabel.innerHTML = 'Salary:';
+salaryLabel.setAttribute('for', 'salary');
+form.append(salaryLabel);
+
+const salaryInput = document.createElement('input');
+
+salaryInput.setAttribute('type', 'number');
+salaryInput.setAttribute('data-qa', 'salary');
+salaryInput.setAttribute('id', 'salary');
+salaryInput.setAttribute('required', '');
+salaryLabel.append(salaryInput);
+
+const notification = document.createElement('div');
+
+notification.classList.add('notification');
+notification.setAttribute('data-qa', 'notification');
+notification.setAttribute('style', 'display: none;');
+
+const notifTitle = document.createElement('div');
+
+notifTitle.classList.add('title');
+notification.append(notifTitle);
+
+const notifDescription = document.createElement('div');
+
+notification.append(notifDescription);
+
+body.append(notification);
+
+const button = document.createElement('button');
+
+button.setAttribute('type', 'submit');
+button.innerHTML = 'Save to table';
+button.classList.add('button');
+form.append(button);
+
+button.addEventListener('click', function (ev) {
+  ev.preventDefault();
+  notification.removeAttribute('style');
+
+  const err1 = nameInput.value.length < 4;
```

Comment:
Use a more descriptive name for error, fix it everywhere
```suggestion
  const nameLengthError = nameInput.value.length < 4;
```